### PR TITLE
fix(phone-input): make the countries searchable

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: src/phone-input/__tests__/phone-input.e2e.js [hunk 1]
```diff
@@ -34,7 +34,14 @@ describe('PhoneInput', () => {
   });
 
   it('passes basic a11y tests', async () => {
-    const accessibilityReport = await analyzeAccessibility(page);
+    const accessibilityReport = await analyzeAccessibility(page, {
```
Comment (gergelyke) [inline src/phone-input/__tests__/phone-input.e2e.js]: I am not sure why it started failing - does anyone have any ideas? 🤔

Comment (chasestarr) [reply src/phone-input/__tests__/phone-input.e2e.js]: can you please share the axe error or remove the rule option so I can view it in CI?

File: src/phone-input/__tests__/phone-input.e2e.js [hunk 2]
```diff
@@ -63,6 +70,23 @@ describe('PhoneInput', () => {
     });
   });
 
+  it('allows a user select a country using the keyboard', async () => {
+    // click select
+    await page.click(selectors.phoneInputSelect);
+    // verify dropdown is open
+    await page.waitFor(selectors.phoneInputSelectDropdown);
+
+    await page.keyboard.type('United Kingdom');
```
Comment (chasestarr) [inline src/phone-input/__tests__/phone-input.e2e.js]: can you please add a test showing that if I type in `United` and press arrow down, I reach `United Arab Emirates`?

File: src/phone-input/__tests__/phone-input.e2e.js (hunk 1 above)
Comment (gergelyke) [reply src/phone-input/__tests__/phone-input.e2e.js]: it was complaining because of this: https://dequeuniversity.com/rules/axe/3.0/label

Comment (chasestarr) [reply src/phone-input/__tests__/phone-input.e2e.js]: I believe you can add an aria-label to the `Input` through overrides

**Comment by houmark** [comment]
This seems to be a boiled down / minimalistic version of the earlier proposed, but never merged PR. This one does not show the text being searched, is that intended? I personally think the other one was much nicer and if someone wants the minimalistic edition like this one, then they could either override or maybe it could be configurable to show or not show a search text field.

I absolutely do not want to slow the merging of this one down as any search improvement is an improvement over the current state of the phone input component.

**Pull request merged**